lib/etc/openturns.conf.in

```diff
@@ -940,9 +940,10 @@
   <GaussianProcessFitter-UnbiasedVariance                       value_bool="true"      />
   <GaussianProcessFitter-UseAnalyticalAmplitudeEstimate         value_bool="true"      />
   <GaussianProcessFitter-DefaultOptimizationLowerBound          value_float="1.0e-2"  />
-  <GaussianProcessFitter-DefaultOptimizationScaleFactor         value_float="2.0"     />
   <GaussianProcessFitter-DefaultOptimizationUpperBound          value_float="1.0e2"   />
   <GaussianProcessFitter-DefaultOptimizationNuggetLowerBound    value_float="1.0e-12"   />
+  <GaussianProcessFitter-OptimizationLowerBoundScaleFactor      value_float="1.0e-2" />
```

Comment:
According to @josephmure :
```suggestion
  <GaussianProcessFitter-OptimizationLowerBoundScaleFactor      value_float="1.0e-3" />
```
Then a test may be to experiment with zero:
```suggestion
  <GaussianProcessFitter-OptimizationLowerBoundScaleFactor      value_float="0.0" />
```
and see what happens.